civiclens/collect/upload_bulk_csvs_to_db.py

```diff
@@ -0,0 +1,178 @@
+from datetime import datetime
+import polars as pl
+import argparse
+
+from civiclens.collect.access_api_data import pull_reg_gov_data
+from civiclens.collect.move_data_from_api_to_database import (
+    insert_comment_into_db,
+)
+from civiclens.utils.constants import REG_GOV_API_KEY
+
+"""
+This code is for one-time and ad hoc bulk uploads of comments.
+Csv files are obtained for known documents here: https://www.regulations.gov/bulkdownload
+Adding comments to the db is faster with this method than via API 
+"""
+
+
+def load_data(file_name: str) -> pl.DataFrame([]):
+    """
+    Loads csv file into a polars df of comments and filters out null comments
+    """
+    q = pl.scan_csv(file_name).filter(~pl.col("Comment").is_null())
+    df = q.collect()
+
+    return df
+
+
+def get_document_objectId(doc_id: str) -> str:
```

Comment:
🔬 `get_document_object_id` is more pythonic. Right now it's a mix of snake and camel case.

Reply:
in this case, objectId is a specific field so i opted to stay closer to the name in the data

Reply:
Makes sense!